src/sqlparser/Cargo.toml

```diff
@@ -27,6 +27,7 @@ normal = ["workspace-hack"]
 [dependencies]
 itertools = { workspace = true }
 serde = { version = "1.0", features = ["derive"], optional = true }
+tokio = { version = "0.2", package = "madsim-tokio" }
```

Comment:
nit: Actually `tokio::task_local` does not rely on any components of the `tokio` runtime. We may consider extracting it into a separate crate in the future.